core/domain/feedback_services.py

```diff
@@ -801,6 +804,19 @@ def _send_feedback_thread_status_change_emails(
                 old_status, new_status)
 
 
+def _check_unique_ids_for_recipients(user_ids, exploration_id, thread_id):
+    user_models = (
+        email_models.FeedbackEmailReplyToIDModel.get_for_multi_user_ids(
+            user_ids, exploration_id, thread_id))
+
+    for user_id in user_ids:
+        if user_models[user_id] is None:
+            user_models[user_id] = (
+                email_models.FeedbackEmailReplyToIDModel.create(
+                    user_id, exploration_id, thread_id))
+            user_models[user_id].put()
```

Comment:
Yeah we can do put_multi() but it won't improve performance because user gets added to feedback thread incrementally. So each time we will look at the list of thread participant only one of them will not have this model. Rest of them should already have it.